### ISSUE TITLE
npm ERR! code EISDIR

### ISSUE THREAD

**Issue by weigeloveu** [PleasureTools/joyBox#18]
`Step 7/32 : RUN npm i     && mv ./node_modules/@types/jsonstream ./node_modules/@types/JSONStream
 ---> Running in b2263074c919
npm ERR! code EISDIR
npm ERR! syscall copyfile
npm ERR! path /app/node_modules/chownr
npm ERR! dest /app/node_modules/.chownr-cyv3QNXu
npm ERR! errno -21
npm ERR! EISDIR: illegal operation on a directory, copyfile '/app/node_modules/chownr' -> '/app/node_modules/.chownr-cyv3QNXu'

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-12-17T13_56_05_099Z-debug.log
The command '/bin/sh -c npm i     && mv ./node_modules/@types/jsonstream ./node_modules/@types/JSONStream' returned a non-zero code: 235
`

**Comment by Snoopycc** [comment]
Unfortunately, I have exactly the same problem. Have you found a solution in the time since?

**Comment by weigeloveu** [comment]
> Unfortunately, I have exactly the same problem. Have you found a solution in the time since?

no,but old version still works,nothing could too do but to wait update

**Comment by judgegc** [comment]
As temporary fix. In `Dockerfile` replace `RUN npm i \
    && mv ./node_modules/@types/jsonstream ./node_modules/@types/JSONStream` with `RUN rm -rf /app/node_modules/chownr \
    && npm i \
    && mv ./node_modules/@types/jsonstream ./node_modules/@types/JSONStream`

**Comment by weigeloveu** [comment]
> As temporary fix. In `Dockerfile` replace `RUN npm i \ && mv ./node_modules/@types/jsonstream ./node_modules/@types/JSONStream` with `RUN rm -rf /app/node_modules/chownr \ && npm i \ && mv ./node_modules/@types/jsonstream ./node_modules/@types/JSONStream`

thanks,its a big help
